node-src/tasks/build.ts

```diff
@@ -60,6 +61,34 @@ export const setBuildCommand = async (ctx: Context) => {
 const timeoutAfter = (ms) =>
   new Promise((resolve, reject) => setTimeout(reject, ms, new Error(`Operation timed out`)));
 
+function isE2EBuildCommandNotFoundError(err) {
+  // It's hard to know if this is the case as each package manager has a different type of
+  // error for this, but we'll try to figure it out.
+  const msg = err.message as string;
+  const errorRegexes = ['command not found', `[\\W]?${e2eBuildBinName}[\\W]? not found`, 'code E404', 'exit code 127'];
+  return errorRegexes.some((regex) => msg.match(new RegExp(regex, 'gi')));
+}
+
+function e2eBuildErrorMessage(err, workingDir, ctx: Context): { exitCode: number, message: string } {
+  const flag = ctx.options.playwright ? 'playwright' : 'cypress';
+
+  // If we tried to run the E2E package's bin directly (due to being in the action)
+  // and it failed, that means we couldn't find it. This probably means they haven't
+  // installed the right dependency or run from the right directory.
+  if (isE2EBuildCommandNotFoundError(err)) {
```

Comment:
@tmeasday as per our slack thread, I fixed this to try to determine if it's the command not found error.